### PR TITLE
Report smoke test failures to #positron-builds, along with initiator name

### PR DESCRIPTION
**Pull request by testlabauto**
Adding slack reporting of smoke test failures on PRs and merges to #positron-builds.  Uses a fork of https://github.com/ivanklee86/xunit-slack-reporter which adds in the name of the user who initiated the workflow.

### QA Notes

All smoke tests pass and results reported to slack.

### CODE REVIEW

File: .github/workflows/positron-merge-to-main.yml [hunk 1]
```diff
@@ -113,3 +113,13 @@ jobs:
         with:
           name: run-artifacts
           path: .build/logs/smoke-tests-electron/
+
+      - name: slack-smoke-test-report
+        if: failure()
+        uses: testlabauto/xunit-slack-reporter@v2.0.1
+        env:
+          SLACK_CHANNEL: C04G3KL70TX
```
Comment (petetronic) [inline .github/workflows/positron-merge-to-main.yml:121]: Should the channels be the same for both? Should we add a comment which channel we're intending to send notifications to?

Comment (testlabauto) [reply .github/workflows/positron-merge-to-main.yml]: Yeah, I was thinking that for both a PR test run failure and a merge to main failure we would report to #positron-builds, but I am open to suggestions.  

Added a commented noting that the channel is #positron-builds.

**Review by petetronic — APPROVED** [review]

**Review by jonvanausdeln — APPROVED** [review]
LGTM!

File: .github/workflows/positron-merge-to-main.yml (hunk 1 above)
Comment (testlabauto) [reply .github/workflows/positron-merge-to-main.yml]: switched to #positron-test-results

**Pull request merged**